archiva-modules/archiva-base/archiva-configuration/src/main/java/org/apache/archiva/configuration/DefaultArchivaConfiguration.java

```diff
@@ -553,10 +554,9 @@ private boolean writeFile(String filetype, String path, String contents) {
      * @param contents the contents to write.
      * @return true if write successful.
      */
-    private boolean writeFile(String filetype, String path, String contents, boolean createDirs) {
-        Path file = Paths.get(path);
-
+    private boolean writeFile(String filetype, String path, String contents, boolean createDirs) {                
         try {
+            Path file = Paths.get(path);
```

Comment:
on windows invalid file could throw InvalidPathException so make it return false.

Reply:
Looks good